molecule/config-values-test/kiali-cr.yaml

```diff
@@ -7,6 +7,7 @@ spec:
   # this test will try to use as many defaults as we can
   istio_namespace: {{ istio.control_plane_namespace }}
   deployment:
+    accessible_namespaces: ["^((?!(istio-operator|kube-.*|openshift.*|ibm.*|kiali-operator)).)*$"]
```

Comment:
This is no longer the default, do you want to keep it as-is?

Reply:
keep it as is - the test actually tests things that expect this. Which is fine - its a good test of the regex anyway.